website/blog/2018-12-21-state-of-react-native-community.md

```diff
@@ -0,0 +1,51 @@
+
+This has been the driving force behind moving away from a monthly release cycle, and the "long term support" approach currently used for version 0.57.x.
+
+Half of the merit in reaching these decisions goes to the other repository created this year:
+
+## [`discussions-and-proposals`](https://github.com/react-native-community/discussions-and-proposals) 🗣
+
+This other ‘meta’ repository, created in July, expanded on the idea of a more open environment for conversations on React Native; while, partially, this need was covered by a few issues labelled [`For Discussion`](https://github.com/facebook/react-native/labels/For%20Discussion) in the main repository, we wanted to give our own spin to the RFC approach that other libraries (like React) have.
+
+This experiment found immediately its role in the React Native lifecycle: even the FB team is now relying heavily on it to discuss what could [be improved in React Native](https://github.com/react-native-community/discussions-and-proposals/issues/64), and coordinate the efforts around the [Lean Core project](https://github.com/react-native-community/discussions-and-proposals/issues/6) - among other interesting conversations.
+
+But we are far from done.
+
+### What awaits ahead 🎓
+
+As explained above, over the past 6-8 months, the [React Native Community organisation](https://github.com/react-native-community) has been used by core contributors to take more ownership over the development of React Native, and enhance the collaboration with Facebook. But this always lacked the formal structure that similar projects may have in place. 
+
+By enforcing a set of standards for all the packages/repos hosted in it, providing a common place for all the maintainers to help each other and providing quality code, this organisation can set the example for everyone in the larger developer community.
+
+We are planning, for the early 2019, to have this new set of guidelines in place - you can let us know what you think in the [dedicated discussion](https://github.com/react-native-community/discussions-and-proposals/issues/63), and in the manifesto proposal that will be written soon.
+
+With this last founding stone, we are confident that the community will become more inclusive and when 1.0 will be reached, we will all be able to write (even more) awesome apps, capable of leveraging this joined effort 🤗
+
+----
+
+I hope you are as excited as I am in looking at the future of this community, and that I will see all of you involved either in the conversations happening in the repositories listed above - or via the awesome code you’ll produce.
```

Comment:
```suggestion
I hope you are as excited as we are about the future of this community. We're excited to see all of you involved either in the conversations happening in the repositories listed above or via the awesome code you’ll produce.
```

- Switch "I" to "we" since the whole article is written in the "we" voice so far
- Add sentence structure